settings.gradle

```diff
@@ -17,6 +17,7 @@ include ':app',
         ':installations',
         ':messaging',
         ':messaging:messaging_java',
+        ':performance',
```

Comment:
I'd also like to ensure that this isn't included in Android builds by default; I think the simplest way is to leave this line commented-out? (You can add a note to the readme about how to enable it.)